cpp/tests/utilities/identify_stream_usage.cpp

```diff
@@ -66,6 +67,34 @@ rmm::cuda_stream_view const get_default_stream()
 }  // namespace test
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🚩 issue:** ‏ The comment above refers to `STREAM_MODE_TESTING=ON` and `STREAM_MODE_TESTING=OFF`, but the code above and the new code you added both use `#ifdef STREAM_MODE_TESTING`. In other words, the code doesn't consider the value but only whether or not `STREAM_MODE_TESTING` is defined. I think the comment above should be rewritten to reflect reality.

Reply:
This is my fault. When I originally wrote that comment I had assigned values, but I ended up just writing the CMake to conditionally define the variable. I agree that updating the comment would be good.

Reply:
Updated the comments for the stream mode testing magic :)